rkt/app_sandbox.go

```diff
@@ -33,6 +34,8 @@ var (
 		Long:  "Initializes an empty pod having no applications.",
 		Run:   runWrapper(runAppSandbox),
 	}
+	flagAnnotations kvMap
```

Comment:
@squeed I think I will stick to kvMap, and add annotation/label one at a time, because I found some values could contain `=` or `,`...